<?xml version="1.0"?>

<launch>
	<arg name="sprayer_robot_name" /> t2
	<arg name="robot_name"/> t1

	<!-- Launch basil detector -->
	<group ns="$(arg sprayer_robot_name)">
		<node name="weed_killer" pkg="assessment_package" type="weed_killer.py" output="screen" args="$(arg sprayer_robot_name)"/>
		<node name="weed_tracker" pkg="assessment_package" type="weed_tracker.py" output="screen" args="$(arg sprayer_robot_name) $(arg robot_name)"/>
	</group>
	
</launch>

